styleguide/haml_locals.rb

```diff
@@ -3,6 +3,120 @@
 # frozen_string_literal => true
 
 @locals = {
+  'table_data' => {
```

Comment:
Not really needed here but we could introduce [faker.js](https://github.com/marak/Faker.js/) at some point to generate mock data - keep it consistent across components + generate data at extremes within parameters.